scipy/interpolate/fitpack2.py

```diff
@@ -1644,8 +1738,23 @@ class RectSphereBivariateSpline(SphereBivariateSpline):
 
     See Also
     --------
-    RectBivariateSpline : bivariate spline approximation over a rectangular
-        mesh
+    BivariateSpline :
+        to create a bivariate spline on a plane grid
+    UnivariateSpline :
+        a similar class for univariate spline interpolation
+    SmoothBivariateSpline :
+        to create a bivariate spline through the given points
+    LSQBivariateSpline :
+        to create a bivariate spline using weighted least-squares fitting
```

Comment:
I'm also not sure about this "to create a ..." clause. Maybe just shorten to "A bivariate spline using ...". Feel free to igore though, it's ESL here :-).

Reply:
I totally agree your comment. So, I shortened the comment.